src/POGOProtos/Map/Pokemon/WildPokemon.proto

```diff
@@ -10,5 +10,5 @@ message WildPokemon {
 	double longitude = 4;
 	string spawn_point_id = 5;
 	.POGOProtos.Data.PokemonData pokemon_data = 7;
-	int32 time_till_hidden_ms = 11;
+	int64 time_till_hidden_ms = 11;
```

Comment:
Okay, it's definitely sent as 32bit already :/ Well I liked negative values more than positive values, that will seem like the real ones.